tuf/utils/utils.go

```diff
@@ -100,10 +110,10 @@ func FindRoleIndex(rs []*data.Role, name string) int {
 // ConsistentName generates the appropriate HTTP URL path for the role,
 // based on whether the repo is marked as consistent. The RemoteStore
 // is responsible for adding file extensions.
-func ConsistentName(role string, hashSHA256 []byte) string {
+func ConsistentName(role data.RoleName, hashSHA256 []byte) data.RoleName {
```

Comment:
This returns a consistent filename for download purposes per TUF's rules, shouldn't be a `data.RoleName` return type.